File: data/java/pattern_7/code.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner createScanner = new Scanner(System.in);
        int num;
        int k = 1;
        System.out.print("Enter the number of rows and columns for the square: ");
        num = createScanner.nextInt();

        for (int i = num; i >= 1; i--) {
            for (int j = num; j >= 1; j--) {
                System.out.print(k + " ");
                k += 2;
            }
            System.out.print("\n");
        }
    }
}

File: data/python/pattern_7/code.py
#MAIN PROGRAM STARTS HERE:
num = int(input('Enter the number of rows and columns for the square: '))
k = 1

for x in range(0, num ):
    for y in range(0, num):
        print ('%d ' % (k), end='')
        k += 2
    print()